TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/Form/CheckboxViewHelper.php

```diff
@@ -18,25 +18,25 @@
  * = Examples =
  *
  * <code title="Example">
- * <typo3.media:form.checkbox name="myCheckBox" value="someValue" />
+ * <neos.media:form.checkbox name="myCheckBox" value="someValue" />
```

Comment:
Even though this is just a comment (used to render documentation!), it will lead people to the use of a not (yet) existing ViewHelper. So, please only merge with the actual vendor name change. Also repeat down below…